incubator/hnc/pkg/controllers/setup.go

```diff
@@ -14,13 +15,20 @@ import (
 
 // Counters of total/current number of hier/object reconciliation attempts.
 var (
-	hcTot int32
-	hcCur int32
-	obTot int32
-	obCur int32
+	hcTot     int32
```

Comment:
These all need to be documented. Also, see the other comments I made about restructuring these.